plugins/standard/applets/conference/conference-calls.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+if(isset($_GET['json'])):
+
+	$ci =& get_instance();
+
+	$service = new Services_Twilio($ci->twilio_sid, $ci->twilio_token);
+
+	$conferences = $service->account->conferences->getIterator(0, 50, array("Status" => "in-progress"));
+
+	$res = array();
+	
+	foreach($conferences as $call) {
+		$res[$call->friendly_name] = array(
+			'date_created' => date("F j, Y, g:i a",strtotime($call->date_created)),
+			'friendly_name' => $call->friendly_name,
+			'status' => $call->status,
+			'duration' => $call->duration
+		);		
+	}
+	
+	die(json_encode($res));
+	
+endif;
+
+?>
+
+<script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jquery/1.2.6/jquery.min.js"></script>
```

Comment:
You should already have access to jQuery here. Do you not? If so, we should fix that problem. Plugins should get access to jQuery by default.